lib/ansible/plugins/action/gather_facts.py

```diff
@@ -13,6 +13,19 @@
 from ansible.plugins.action import ActionBase
 from ansible.utils.vars import merge_hash
 
+# Connection and fact module pairs in 2.8's CONNECTION_FACTS_MODULES
```

Comment:
+1 to maybe just pulling out the legacy networking exception cases for this, since that's mostly for really old stuff that's no longer being actively developed- would make the code a lot simpler for a legacy corner case that's likely not being used since all the active development is on the collection versions.

Reply:
Done :zap: